app/views/groups/show.html.erb

```diff
@@ -0,0 +1,26 @@
+<div class="transaction">
+<div class="card mb-4">
+<div class="card-body amount-card">
+<p>Total Amount</p>
+<p class='tot-amount'>$ <%= @group.total_amount %></p>
+</div>
+</div>
+<% if @entities.any? %>
+  <% @entities.each do |entity| %>
+    <div class="card mb-4">
+      <div class="card-body amount-card">
+        <p class="cat-name"><%= entity.name %></p>
+        <p class="cat-amount">$<%= entity.amount %></p>
+      </div>
+    </div>
+  <% end %>
+<% else %>
+  <div>
+    <p colspan="2" class="text-center border text-gray-600 p-1">No Transaction.</p>
+  </div>
+<% end %>
+<div class="d-flex justify-content-center gap-2">
+  <%= link_to "New Transaction", new_group_entity_path(@group), class: "btn btn-outline-success"%>
+  <%= link_to "Back", groups_path, class: "btn back-btn btn-outline-success" %>
+</div> 
+</div>
```

Comment:
**OPTIONAL** - You have done well so far, but please note that according to the project requirements, you are to style your project according to the given design, doing this would give confidence to potential recruiters in your ability to replicate any design they require you to. Please style this page as it is in the given design.

Your design:
![erfgqwrgfqwrf](https://github.com/manzitresor/budget_app/assets/73607512/9b5eb8a2-2db8-46e1-ad4a-92dad2f41ad1)

DesignTemplate:
![Cwrgawrfgawr](https://github.com/manzitresor/budget_app/assets/73607512/3eec7e27-47ec-4a05-b4c9-e1ee20178d21)